README.md

```diff
@@ -58,18 +68,251 @@ Check out the [configuration part](https://docs.thundra.io/python/configuration-
+- For decorator tracing method, just change `@thundra.<framework_wrapper>` by `@thundra.django_wrapper`.
+
+
+### Flask
+
+Tracing Flask application:
+
+1. Auto trace by using one of the following methods:
+    - [No-Code Change Tracing](#no-code-change-tracing).
+    - [In-Code Configuration Tracing](#in-code-configuration-tracing)
+2. Trace specific endpoints by [Decorator](#decorator)
+
+Code snippet in [In-Code Configuration Tracing](#in-code-configuration-tracing) should be added into `app.py` file before
+application is initialized.
+
+**NOTES**
+
+- Make sure to choose just one of the methods.
+- For decorator tracing method, just change `@thundra.<framework_wrapper>` by `@thundra.flask_wrapper`.
+
+### Fastapi
+
+Tracing Fastapi application:
+
+1. Auto trace by using one of the following methods:
+    - [No-Code Change Tracing](#no-code-change-tracing).
+    - [In-Code Configuration Tracing](#in-code-configuration-tracing)
+2. Trace specific endpoints by [Decorator](#decorator)
+
+
+**NOTES**
+
+- Make sure to choose just one of the methods.
+- For decorator tracing method, just change `@thundra.<framework_wrapper>` by `@thundra.fastapi_wrapper`.
+- Fastapi has been supported for python 3.7 and above.
+
+### Chalice
+
+Tracing Fastapi application:
+
+1. Auto trace by using one of the following methods:
+    - [No-Code Change Tracing](#no-code-change-tracing).
+    - [In-Code Configuration Tracing](#in-code-configuration-tracing)
+2. Trace specific endpoints by [Decorator](#decorator)
+
+
+**NOTES**
+
+- Make sure to choose just one of the methods.
+- For decorator tracing method, just change `@thundra.<framework_wrapper>` by `@thundra.fastapi_wrapper`.
+
+
+## Integrations
+
+Thundra provides out-of-the-box instrumentation (tracing) for following libraries.
+
+|Library             |Supported Version          |
+|--------------------|---------------------------|
+|logging             |Fully supported            |
+|requests            |`>=2.0.0`                  |
+|redis               |`>=2.10.0`                 |
+|pymongo             |`>=3.0.0`                  |
+|PyMySQL             |`>=0.7.0`                  |
+|MySQLdb             |`>=1.0.0`                  |
+|psycopg2            |`>=2.2.0`                  |
+|botocore (boto3)    |`>=1.4.0`                  |
+|SQLAlchemy          |`>=1.2.0`                  |
+|elasticsearch       |`>=0.4.1`                  |
+|aiohttp             |`>=1.1.5`                  |
 
-Instead, you can [setup async monitoring](https://apm.docs.thundra.io/performance/zero-overhead-with-asynchronous-monitoring) in **2 minutes** and monitor your lambda functions with **zero overhead**!
 
 ## Log Plugin
 Log plugin is added by default, but in order to enable it, you should add `ThundraLogHandler` to your logger.
 
 You can either add it via logging.conf or add during getting logger object.
 
-#### Configurepy tes via logging.conf
+#### Configurepy test via logging.conf
```

Comment:
"Configurepy test via logging.conf" should be "Configure via logging.conf"

Reply:
done.